src/assets/TokensController.test.ts

```diff
@@ -19,10 +21,21 @@ describe('TokensController', () => {
       onPreferencesStateChange: (listener) => preferences.subscribe(listener),
       onNetworkStateChange: (listener) => network.subscribe(listener),
     });
+
+    clock = sinon.useFakeTimers(1);
```

Comment:
What is the clock used for?

Reply:
mocking date.now() for this test: https://github.com/MetaMask/controllers/blob/74e62f1c958e1b87435a6339bd97ade8b28ede90/src/assets/TokensController.test.ts#L501

Reply:
Thanks, that makes sense. It would be helpful if there was a comment on that line referencing the clock though, or even just that it relies upon fake timers. I had looked for a usage of that variable and found none, and thought it was unused. Timing stuff can be pretty subtle.

Reply:
Done!

Reply:
And actually just moved the fakeTimers call into the one test that it uses for improved visibility/clarity.

Reply:
Nice! Thanks, this is much better.